### PR TITLE
Website: Responsify top pages from header & footer, /about, /careers-in-tech

### PR DESCRIPTION
**Pull request by tanyaparker**

### CODE REVIEW

**Comment by tanyaparker** [comment]
cc @breville

**Comment by breville** [comment]
should we cull the (restored) pages?

File: pegasus/sites.v3/code.org/public/action/AL.md [hunk 1]
```diff
@@ -1,80 +0,0 @@
----
```
Comment (Erin007) [inline pegasus/sites.v3/code.org/public/action/AL.md:1]: Why remove all of the action/state files?

Comment (tanyaparker) [reply pegasus/sites.v3/code.org/public/action/AL.md:1]: We don't use these anymore. These were replaced by the state pages on /promote.

**Comment by tanyaparker** [comment]
@breville I don't understand your comment. What do you mean?

**Pull request merged**